raiden_contracts/data_1.0.0/source/services/UserDeposit.sol

```diff
@@ -6,9 +6,9 @@ import "raiden/Token.sol";
 import "raiden/Utils.sol";
 
 contract UserDeposit is Utils {
-    uint constant public withdraw_timeout = 15 * 100;  // time before withdraw is allowed in seconds
+    uint public withdraw_timeout;  // time before withdraw is allowed in seconds
```

Comment:
Maybe this should be made `immutable`? Would still allow setting it in constructor, but better inform it can't be changed latter

Reply:
Yes, we could do this at a dozen places in the contracts. We have plenty of contract properties that get set by the constructor exclusively. But the gain of doing so is very little. Putting this into the context of where we are in the project right now, I don't think it is worth the time and effort. Should be a deployment only advantage, if I'm not mistaken.

Reply:
Nevertheless this would be another PR and definitely not part of this one. Therefore I would recommend to create a new issue from this and close the conversation. We can then pull in this new issue any time. Though it isn't my PR, nor my review comment thread, so this is just my strong recommendation.